src/js/libcs/Accessors.js

```diff
@@ -168,6 +168,24 @@ Accessor.setField = function(geo, field, value) {
         movepointscr(geo, List.turnIntoCSList([value.value[0], value.value[1], CSNumber.real(1)]), "homog");
     }
 
+    if (field === "x" && geo.kind === "P" && geo.movable && (List._helper.isNumberVecN(value, 1) || value.ctype === "number")) {
+        if (CSNumber._helper.isAlmostZero(geo.homog.value[2])) {
+            console.log("cannot set x component without y component of far points");
```

Comment:
Cinderella treats assignment of `A.x` in an interesting way:

```
A.homog=[1,2,0];
A.x = 2;
err (A.homog);
```

yields

`A.homog ==> [4,0,0]`

Maybe @richter or @kortenkamp can comment on the issue?